build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -17,20 +17,18 @@ if($Help)
     exit 0
 }
 
-$Architecture='x64'
-
 $RepoRoot = "$PSScriptRoot\..\.."
 $AppPath = "$PSScriptRoot"
 
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows"
```

Comment:
The main point of the comment for this file is: 

"the INSTALL dir between update-dependencies and init-tools should be the same place, so that way when we run them on our dev boxes, we don't download stage0 twice."

Whever the "build" command puts Stage0, the "update-dependencies" command should also put Stage0.